buildspec.yml

```
@@ -1,11 +1,29 @@
 version: 0.2
 
+env:
+  variables:
+    SMTP_CREDS_SECRET: cfpb/team/regtech/smtp-ses-creds
+    SMTP_FROM_ADDRESS: no-reply@cfpb.gov
```

For whatever reason, emails from `no-reply` get filtered for me. :shrug: 

```suggestion
    SMTP_FROM_ADDRESS: noreply@cfpb.gov
```